charts/studio/values.yaml

```
@@ -8,6 +8,8 @@ imagePullSecrets: []
 global:
   # -- Studio: Hostname for accessing Studio (no http(s) scheme)
   host: "studio.example.com"
+  # -- Studio: Base path (prefix)
+  basePath: ""
```

When I set it to `"/"` locally, I wasn't still able to access the static resources. I would add condition  to handle empty `""` string and  slash`"/"` 
```suggestion
  # -- Studio: Base path (prefix; default: "/")
  basePath: ""
```